tmexp/metrics.py

```diff
@@ -24,6 +24,21 @@ def metric_stats(metric: np.array, num_tabs: int, logger: Logger) -> None:
         logger.info("%s%s : %.2f", num_tabs * "\t", op_name, op(metric))
 
 
+def compute_distinctness(
+    wordtopic: np.array, num_topics: int, num_words: int
+) -> np.array:
+    distinctness = np.zeros((num_topics, num_topics))
+    wordtopic += 1 / num_words
+    wordtopic /= np.sum(wordtopic, axis=1)[:, None]
+    for i in range(num_topics):
+        dist_i = wordtopic[i, :]
+        for j in range(num_topics):
```

Comment:
for dist_j in wordtopic
```suggestion
        for dist_j in wordtopic:
```

Reply:
done

Reply:
same as above